api/app.py

```diff
@@ -194,3 +194,52 @@ def get_assignment_id(category_type: str, assignment_number: int, lab_type: int
 
     # Return assignment ID if found for categories other than "labs"
     return {"assignment_id": assignment_data.get("assignment_id", "Assignment ID not found.")}
+
+
+@app.get("/FetchAllGrades")
+@handle_errors
+def fetchAllGrades(class_id: str = None):
+    """
+    Fetch Grades for all assignments for all students
+
+    Parameters:
+    - class_id (str, optional): The ID of the class for which assignments are being retrieved. 
+      Defaults to `None`.
+
+    Returns:
+    - JSON
+
+    Example Output:
+    {
+        "5211610": [
```

Comment:
Do we want to store the assignment ID as the key instead? Or maybe another ticket is to create this GCP SQL database of assignmentIDs -> title of assignment, similar to Naveen's script. 

@mananb77

Reply:
Yes, we need to create the SQL database of assignment ID --> title so we can use ID for assignment mapping instead of naming for the future.